python/xorq/ibis_yaml/tests/test_compiler.py

```diff
@@ -89,11 +89,11 @@ def test_ibis_compiler_parquet_reader(build_dir):
     assert expr.execute().equals(roundtrip_expr.execute())
 
 
-def test_compiler_sql(build_dir):
+def test_compiler_sql(build_dir, parquet_dir):
     backend = xo.datafusion.connect()
     awards_players = deferred_read_parquet(
         backend,
-        xo.config.options.pins.get_path("awards_players"),
+        str(parquet_dir / "awards_players.parquet"),
```

Comment:
why are we casting to `str`?

Reply:
Because the yaml compiler does not know how to handle Path